src/entrypoint/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+)
+
+const listenPort = "8080"
+const providerId = "plesk"
+const providerName = "Plesk"
+const providerDisplayName = "Domain powered by Plesk"
+const windowWidth = 750
+const windowHeight = 750
+
+type response struct {
+	ProviderId          string `json:"providerId"`
+	ProviderName        string `json:"providerName"`
+	ProviderDisplayName string `json:"providerDisplayName"`
+	UrlSyncUX           string `json:"urlSyncUX"`
+	UrlAPI              string `json:"urlAPI"`
+	Width               int    `json:"width"`
+	Height              int    `json:"height"`
+}
+
+var reUrl *regexp.Regexp
+
+func init() {
+	reUrl = regexp.MustCompile(`(?:/host/(?P<host>.*?))?(?:/port/(?P<port>\d*?))?/v2/(?P<domain>.*?)/settings(/|$)`)
+}
+
+func main() {
+	log.Printf("Start server listening on %s port...", listenPort)
+	http.HandleFunc("/", handleEntryPoint)
+
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", listenPort), nil))
+}
+
+func handleEntryPoint(w http.ResponseWriter, r *http.Request) {
+	url := r.URL.Path
+	host, port, err := parseUrl(url)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		log.Printf("Failed to parse URL '%s': %s", url, err)
+		return
+	}
+
+	entryPointUrl := fmt.Sprintf(`https://%v:%v/modules/domain-connect/index.php/`, host, port)
+	res := response{
+		ProviderId:          providerId,
+		ProviderName:        providerName,
+		ProviderDisplayName: providerDisplayName,
+		UrlSyncUX:           entryPointUrl,
+		UrlAPI:              entryPointUrl,
+		Width:               windowWidth,
+		Height:              windowHeight,
+	}
+
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+	if err := json.NewEncoder(w).Encode(res); err != nil {
+		panic(err)
```

Comment:
Can you please also replace this `panic()` statement by return of `http.StatusServiceUnavailable`? And log the error.

After panic program would be terminated, but in this case it is sufficient to skip the problematic request and continue to serve others.